shell/edit/workload/storage/index.vue

```diff
@@ -55,41 +54,32 @@ export default {
       type:    Array,
       default: () => [],
     },
+    namespacedPvcs: {
+      type:    Array,
+      default: () => [],
+    },
 
     registerBeforeHook: {
       type:    Function,
       default: null,
     },
-  },
-
-  async fetch() {
-    if ( this.$store.getters['cluster/schemaFor'](PVC) ) {
-      this.pvcs = await this.$store.dispatch('cluster/findAll', { type: PVC });
-    } else {
-      this.pvcs = [];
-    }
+    asyncDataLoading: {
+      default: false,
+      type:    Boolean
+    },
   },
 
   data() {
     this.initializeStorage();
 
-    return {
-      pvcs:           [],
-      storageVolumes: this.getStorageVolumes(),
-    };
+    return { storageVolumes: this.getStorageVolumes() };
   },
 
   computed: {
     isView() {
       return this.mode === _VIEW;
     },
 
-    namespacedPVCs() {
```

Comment:
This filter isn't applied to the list that's passed in (`:namespaced-pvcs="pvcs"`) --> `[PVC]:             { applyTo: [{ var: 'pvcs' }] },`